src/manage/client-list.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react'
 
-import { Heading1 } from '../components'
+import { PagerHeader } from '../components'
 
 class ClientList extends React.Component {
   render() {
     return (
       <div>
-        <Heading1>Clients</Heading1>
-        <div>client list</div>
-        <br />
-        <br />
+        <PagerHeader
+          title="Clients"
+          description={{
+            text: 'Setup a mobile, web or IoT application to use Auth0 for Authentication.',
+            learnMore: '/clients'
```

Comment:
@landitus Can we freeze this in the API?

I like how this looks, but not sure if it fits all the use cases

Reply:
Yep, I think we're good 👍